web-registry/src/components/molecules/AdditionalMetadata.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import { Box, Collapse } from '@mui/material';
+import { useTheme } from '@mui/styles';
+import Title from 'web-components/lib/components/title';
+import { ExpandButton } from 'web-components/lib/components/buttons/ExpandButton';
+import { formatDate } from 'web-components/lib/utils/format';
+import { Theme } from 'web-components/lib/theme/muiTheme';
+
+import { LinkWithArrow } from '../atoms';
+import { LineItemLabelAbove } from '../molecules';
+import { VcsProjectMetadata } from '../../types/project/project';
+
+export interface MetadataProps {
+  metadata: VcsProjectMetadata;
+  creditClass: any;
+  creditClassVersion: any;
+  additionalCertification: any;
+}
+
+const AdditionalMetadata: React.FC<MetadataProps> = ({
+  metadata,
+  creditClass,
+  creditClassVersion,
+  additionalCertification,
+}) => {
+  const theme = useTheme<Theme>();
```

Comment:
I believe if you import `@mui/material` (instead of `@mui/styles`), you shouldn't have to pass the theme type